docs/en_US/Blog/ParallelizingSequentialSearch.md

```diff
@@ -0,0 +1,109 @@
+# Parallelizing Sequential Search
```

Comment:
Parallelizing Sequential Search for SMOB / TPE?

Reply:
Done.